lighthouse-core/gather/driver/execution-context.js

```diff
@@ -97,7 +97,8 @@ class ExecutionContext {
             .catch(${pageFunctions.wrapRuntimeEvalErrorInBrowserString})
             .then(resolve);
         });
-      }())`,
+      }())
+      //# sourceURL=lighthouse-eval.js`,
```

Comment:
lets give them unique names since we have so many of these.

how about

```js
//# sourceURL=lighthouse_eval_${this.getContextId()}_${++i}.js`,
```

Reply:
This would require a regex elsewhere to exclude. Is that worth it? How do you see yourself making use of these things which we wish to drop from our artifacts being unique?